model/Core/Vocabularies/RelationshipType.md

```diff
@@ -44,16 +44,16 @@ between a Package and a File, between two Packages, or between one SPDXDocument
 - example: Every `to` Element is an example for the `from` Element
 - evidenceFor: (Dataset) Every `to` Element is can be considered as evidence for the `from` Element
 - expandedFromArchive: Every `to` Element is an artifact expanded from the `from` archive file
-- exploitCreatedBy: (Security) Designates an agent has created an exploit against a vulnerability
+- exploitCreatedBy: (Security) The `to` Agent(s) has created an exploit against a `from` Vulnerability element
```

Comment:
The `from` Vulnerability has had an exploit created against it by each `to` Agent